{"kind":"symbol","schemaVersion":{"patch":0,"major":0,"minor":3},"relationshipsSections":[{"title":"Conforms To","identifiers":["doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/s5ErrorP","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/s8SendableP"],"type":"conformsTo","kind":"relationships"}],"metadata":{"externalID":"s:27PasswordAndNfinityFramework0C10CloudErrorO","modules":[{"name":"PasswordAndNfinityFramework"}],"symbolKind":"enum","role":"symbol","title":"NfinityCloudError","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"NfinityCloudError","kind":"identifier"}],"navigatorTitle":[{"kind":"identifier","text":"NfinityCloudError"}],"roleHeading":"Enumeration"},"hierarchy":{"paths":[["doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework"]]},"primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["Mac Catalyst"],"tokens":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"kind":"identifier","text":"NfinityCloudError"}]}],"kind":"declarations"}],"abstract":[{"type":"text","text":"Represents an enum hierarchy for errors related to the Nfinity Cloud."}],"topicSections":[{"title":"Enumeration Cases","identifiers":["doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/attemptExceeded","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/cloudError(errorCode:errorMessage:)","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/cloudHTTPError(httpCode:response:)","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/httpDeserialization(httpCode:cause:headers:)","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/network(cause:)"]},{"title":"Default Implementations","identifiers":["doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/Error-Implementations"],"generated":true}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/passwordandnfinityframework\/nfinityclouderror"]}],"identifier":{"url":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError","interfaceLanguage":"swift"},"sections":[],"references":{"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/Error-Implementations":{"type":"topic","title":"Error Implementations","role":"collectionGroup","url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/error-implementations","kind":"article","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/Error-Implementations","abstract":[]},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/cloudHTTPError(httpCode:response:)":{"url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/cloudhttperror(httpcode:response:)","abstract":[{"type":"text","text":"Represents an HTTP error from the Nfinity Cloud with the associated HTTP status code and a custom response object."}],"fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"cloudHTTPError"},{"text":"(","kind":"text"},{"text":"httpCode","kind":"externalParam"},{"kind":"text","text":": "},{"text":"Int","preciseIdentifier":"s:Si","kind":"typeIdentifier"},{"kind":"text","text":", "},{"kind":"externalParam","text":"response"},{"text":": ","kind":"text"},{"text":"NfinityCloudErrorResponse","kind":"typeIdentifier","preciseIdentifier":"s:27PasswordAndNfinityFramework0C18CloudErrorResponseV"},{"kind":"text","text":")"}],"title":"NfinityCloudError.cloudHTTPError(httpCode:response:)","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/cloudHTTPError(httpCode:response:)","role":"symbol","type":"topic","kind":"symbol"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework":{"abstract":[],"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework","type":"topic","url":"\/documentation\/passwordandnfinityframework","role":"collection","kind":"symbol","title":"PasswordAndNfinityFramework"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/s8SendableP":{"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/s8SendableP","title":"Swift.Sendable","type":"unresolvable"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/httpDeserialization(httpCode:cause:headers:)":{"url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/httpdeserialization(httpcode:cause:headers:)","abstract":[{"text":"Indicates a problem with deserializing the HTTP response into a model object.","type":"text"}],"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"httpDeserialization"},{"kind":"text","text":"("},{"text":"httpCode","kind":"externalParam"},{"kind":"text","text":": "},{"text":"Int","preciseIdentifier":"s:Si","kind":"typeIdentifier"},{"text":", ","kind":"text"},{"kind":"externalParam","text":"cause"},{"kind":"text","text":": "},{"text":"Error","kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP"},{"text":", ","kind":"text"},{"text":"headers","kind":"externalParam"},{"text":": [","kind":"text"},{"preciseIdentifier":"s:s11AnyHashableV","text":"AnyHashable","kind":"typeIdentifier"},{"text":" : ","kind":"text"},{"text":"Any","kind":"keyword"},{"text":"])","kind":"text"}],"title":"NfinityCloudError.httpDeserialization(httpCode:cause:headers:)","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/httpDeserialization(httpCode:cause:headers:)","role":"symbol","type":"topic","kind":"symbol"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/s5ErrorP":{"type":"unresolvable","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/s5ErrorP","title":"Swift.Error"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError":{"role":"symbol","type":"topic","navigatorTitle":[{"text":"NfinityCloudError","kind":"identifier"}],"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"NfinityCloudError"}],"url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror","title":"NfinityCloudError","kind":"symbol","abstract":[{"type":"text","text":"Represents an enum hierarchy for errors related to the Nfinity Cloud."}]},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/FaceDetectionSender/sendFaceDetection(imageData:)":{"url":"\/documentation\/passwordandnfinityframework\/facedetectionsender\/sendfacedetection(imagedata:)","abstract":[{"type":"text","text":"Method for sending face detection data, and returns an "},{"code":"Swift.Result","type":"codeVoice"},{"text":" instance.","type":"text"}],"fragments":[{"text":"func","kind":"keyword"},{"text":" ","kind":"text"},{"text":"sendFaceDetection","kind":"identifier"},{"text":"(","kind":"text"},{"text":"imageData","kind":"externalParam"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:10Foundation4DataV","text":"Data"},{"text":") ","kind":"text"},{"text":"async","kind":"keyword"},{"kind":"text","text":" -> Swift"},{"text":".","kind":"text"},{"kind":"typeIdentifier","text":"Result","preciseIdentifier":"s:s6ResultO"},{"kind":"text","text":"<"},{"text":"FaceDetectionState","kind":"typeIdentifier","preciseIdentifier":"s:27PasswordAndNfinityFramework18FaceDetectionStateO"},{"text":", ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:27PasswordAndNfinityFramework0C10CloudErrorO","text":"NfinityCloudError"},{"text":">","kind":"text"}],"title":"sendFaceDetection(imageData:)","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/FaceDetectionSender\/sendFaceDetection(imageData:)","role":"symbol","type":"topic","kind":"symbol"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/attemptExceeded":{"url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/attemptexceeded","abstract":[{"text":"Case represents too many tries on ","type":"text"},{"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/FaceDetectionSender\/sendFaceDetection(imageData:)","isActive":true,"type":"reference"},{"text":".","type":"text"}],"fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"attemptExceeded"}],"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/attemptExceeded","title":"NfinityCloudError.attemptExceeded","role":"symbol","type":"topic","kind":"symbol"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/cloudError(errorCode:errorMessage:)":{"type":"topic","kind":"symbol","url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/clouderror(errorcode:errormessage:)","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/cloudError(errorCode:errorMessage:)","role":"symbol","title":"NfinityCloudError.cloudError(errorCode:errorMessage:)","abstract":[{"text":"Represents an error with a specific error code and message that the Nfinity Cloud might return, but not from network call.","type":"text"}],"fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"cloudError","kind":"identifier"},{"kind":"text","text":"("},{"kind":"externalParam","text":"errorCode"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"text":"?, ","kind":"text"},{"text":"errorMessage","kind":"externalParam"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?)"}]},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/network(cause:)":{"title":"NfinityCloudError.network(cause:)","type":"topic","abstract":[{"text":"Case represents network error","type":"text"}],"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/network(cause:)","kind":"symbol","role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"network"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"cause"},{"kind":"text","text":": "},{"text":"CommunicatorError","kind":"typeIdentifier","preciseIdentifier":"s:21CommunicatorFramework0A5ErrorO"},{"text":")","kind":"text"}],"url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/network(cause:)"}}}